### ISSUE TITLE
Power consumption down but hashrate remains the same

### ISSUE THREAD

**Issue by Softybeast** [trexminer/T-Rex#471]
Recently I updated my Nvidia drivers after I read about the RTX3080 and 3090 getting mining nerfs. After installing the drivers, my power consumption went from 320W to 260W and my hashrate remained the same at 86MH/s. This is not really a concern, but surely something is not right, after rerolling the drivers, it went back to 320w, but a day later(now) it is back at 260W.... Is this a concern or should I just be happy that my power consumption is down significantly?

**Comment by pdieppa** [comment]
I have been upgrading my Nvidia drivers when new ones come out and have not see any changes to the power. I don't have any 3080 or 3090 so I wonder if the changes are specific to those cards. I have 3060tis and 3070s but the power has not changed. I'm up to 465.24.02 (linux). How are you managing the memory and GPU? 320w sounds like default settings and 260w after some OC settings have been applied.

**Comment by Softybeast** [comment]
No,  there hasnt been any changes. It must have been some OC that for some reason kicked in after the update. I woke up this morning and the power consumption was back at 320W so I still dont really know what happened

**Comment by pdieppa** [comment]
If have seen that happened when MSI Afterburner crashes or it resets itself, applying the default values to the card. 
If you are indeed using Windows, there is a way that you can run Afterburner from the command line. This allows you to schedule a command to run at a set time, or a recurring event like once every hour making sure your card always has the correct settings. I use HiveOS and it manages all that for me.

**Comment by fedaykinofdune** [comment]
> I have seen that happened when MSI Afterburner crashes or it resets itself, applying the default values to the card.
I've had the same thing happen

> If you are indeed using Windows, there is a way that you can run Afterburner from the command line. 
I just discovered that overclocking works better when you let t-rex manage it. Use the config.example file as your template, set your pool urls and credentials in it, and set your core clocks with the "cclock" and "mclock" variables, making sure to use the offsets rather than the actual value. I run my core clock -200 mhz so I set cclock to -200. Also, when using afterburner for the OC, the core clock would still jump up and down anyway, so the lock-cclock variable in the config is very helpful. That one takes the actual value so with my cclock -200 I lock it with lock-cclock at 1050. I immediately noticed that I'm getting 1-2mh more than when using afterburner, possibly since the clock rate isn't jumping around anymore. 

Once you have your config file set up properly, make sure you set t-rex.exe to run with admin rights, and all you need in the .bat file is:
t-rex.exe --config your_config_filename

Takes a bit of work getting it all set up the way you want but definitely worth it!
